go.mod

```diff
@@ -59,7 +60,7 @@ require (
 	k8s.io/kube-state-metrics v1.7.2
 	k8s.io/kubectl v0.0.0
 	k8s.io/kubernetes v1.16.2 // indirect
-	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/controller-runtime v0.5.1
```

Comment:
Why upgrade the versions?
It would need to be done in another PR.

Reply:
reverted